export/tests/lsblk_sample.py

```diff
@@ -84,6 +84,25 @@
     ],
 }
 
+WHOLE_DEVICE_VC_WRITABLE = {
```

Comment:
Confirming that this is roughly the same output I see when I run `lsblk --json --output NAME,RO,TYPE,MOUNTPOINT,FSTYPE`.